H_ecto_models.md

```diff
@@ -561,20 +567,17 @@ defmodule HelloPhoenix.User do
 end
 ```
 
-And here is how we would declare the complementary `belongs_to` relationship in `web/models/video.ex`:
+Since we used generator to scaffold our `Video` model and specified user association with `user_id:references:users` `belongs_to` relationship will already be defined for us in `web/models/video.ex`:
```

Comment:
I think this is great. The only thing I would change is to add some definite articles:

Since we used the generator . . . and specified the user association with `user_id:references:users`, the `belongs_to` . . .